src/index.js

```diff
@@ -66,14 +66,15 @@ export default class VueI18n {
 
     const locale: Locale = options.locale || 'en-US'
     const fallbackLocale: Locale = options.fallbackLocale || 'en-US'
-    const messages: LocaleMessages = options.messages || {}
     const dateTimeFormats = options.dateTimeFormats || {}
     const numberFormats = options.numberFormats || {}
+    const prefix: Object = this.prefixedConstructor(options.prefix)
+    const messages: LocaleMessages = this.createPrefixedMessages(prefix, options.messages)
 
     this._vm = null
     this._formatter = options.formatter || defaultFormatter
     this._modifiers = options.modifiers || {}
-    this._missing = options.missing || null
+    this._missing = options.missing === undefined ? this.createPrefixedMissingHandler(prefix) : options.missing
```

Comment:
This changes default missing handler from `null` (no handler) to new one.
I am afraid that it may be a breaking change to surpise users by showing icons in their production. (Ofcourse they should update it first on test environment though)

So how about to keep `missing` as `null` and new prefix handlers should be explicitly used ?

Reply:
createPrefixedMissingHandler will return null if no prefix

Reply:
Ah, understood.
But if so, I think adding`prefix` option just for `createPrefixedMissingHandler` complicate code base.
`prefix` option are not used if user passed their own `missing` option.

Reply:
should it throw a warn if both are set?

Reply:
on the docs it could show like this: 

```
i18n = new VueI18n({
        locale: 'en',
        fallbackLocale: 'en',
        messages,
        prefix: {
          active: process.env.NODE_ENV !== 'production',
          translated,
          untranslated
        }
      })

```